app/src/Routes/Works/Works.js

```diff
@@ -1,9 +1,26 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
+import * as works from './works.json';
+
+const Piece = (piece) => (
+    <section>
+        <div>{piece.title}</div>
+        <div>{piece.type}</div>
+        {piece.instruments ? piece.instruments.join(",") : "" }
+        <a href={piece.detailLink} target={piece.external ? "_blank" : ""}>Details</a>
+    </section>
+)
+
+const WorksCategory = (categoryName, category) => (
+    <figure className="category">
```

Comment:
`figure` doesn't really make sense here. `figure` is typically used for images w/ an optional caption. but i'm also don't think nesting `section` or `list` elements within the `figure` element is valid

Reply:
sounds good, switched to an li